controllers/permits/applications/special-uses/commercial/outfitters/index.js

```diff
@@ -110,9 +120,48 @@ const post = function(req, res){
 		const response = include('test/data/outfitters.post.json');
 
 		response.apiRequest = postData;
-    
-		res.json(response);
-    
+
+		// api database updates
+		const controlNumber = Math.floor((Math.random() * 10000000000) + 1);
```

Comment:
Might be good to check for a collision on the controlNumber, and handling it if there is one.